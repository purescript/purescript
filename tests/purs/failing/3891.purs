-- @shouldFailWith TypesDoNotUnify
module Main where

oops = "(" ")"
