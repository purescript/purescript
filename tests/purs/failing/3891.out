Error found:
in module [33mMain[0m
at tests/purs/failing/3891.purs:4:8 - 4:15 (line 4, column 8 - line 4, column 15)

  Could not match type
  [33m        [0m
  [33m  String[0m
  [33m        [0m
  with type
  [33m              [0m
  [33m  String -> t0[0m
  [33m              [0m

while applying a function [33m"("[0m
  of type [33mString[0m
  to argument [33m")"[0m
while inferring the type of [33m"(" ")"[0m
in value declaration [33moops[0m

where [33mt0[0m is an unknown type

See https://github.com/purescript/documentation/blob/master/errors/TypesDoNotUnify.md for more information,
or to contribute content related to this error.

